### PR TITLE
Updated Rate Limits for Dev (Paid) and One-App Columns in Tables

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Updated rate limit tables
- **Is this PR related to a Monolith release?** Yes, Release 2021.10.0

### Resolves:

* [OKTA-433814](https://oktainc.atlassian.net/browse/OKTA-433814)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ This page provides the API rate limits for management activities, which is part
...
+| **Create or list users:**<br>Only `GET` or `POST` to `/api/v1/users`                                                               | 100              | 600              | 600     | 600        | 600                   |
+| **Update or delete a user by ID:**<br>Only `POST`, `PUT` or `DELETE` to `/api/v1/users/${id}`                                       | 100              | 600              | 600     | 600        | 600                   |
+| **Get System Log data:**<br>`/api/v1/logs`                                                                                         | 20               | 50               | 50      | 50         | 120                   |
+| **Get System Log data:**<br>`/api/v1/events`                                                                                       | 20               | 50               | 50      | 50         | 100                   |
+| **OAuth2 client configuration requests:**<br>`/oauth2/v1/clients`                                                                  | 25               | 100               | 100      | 100        | 100                   |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md:33]: Do we need to change "OAuth2" to "OAuth 2.0"?

**Review by vanngo-okta — APPROVED** [review]
Just a question about style. Otherwise, LGTM! 🚀

File: packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md:33]: I did a search in dev docs and noticed that when they're in an endpoint, they're `/oauth2`.

**Review by haozhao-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md [hunk 2]
```diff
@@ -21,17 +21,17 @@ This page provides the API rate limits for management activities, which is part
...
-| **Get, update, or delete an application by ID:**<br>`/api/v1/apps/${id}` only                                                       | 100              | 300              | 300     | 600        | 500                   |
+| **Cumulative rate limit**                                                                                                         | **980**          | **5,200**        | **5,200**| **5,200** | **7,000**             |
+| **Create or list applications:**<br>`/api/v1/apps` except `/api/v1/apps/${id}`                                                      | 20               | 100               | 100      | 100        | 100                   |
+| **Get, update, or delete an application by ID:**<br>`/api/v1/apps/${id}` only                                                       | 100              | 600              | 600     | 600        | 500                   |
 | **Create or list groups:**<br>`/api/v1/groups` except `/api/v1/groups/${id}`                                                        | 100              | 300              | 300     | 600        | 500                   |
```
Comment (haozhao-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md:27]: this one also needs the change

**Review by joec-okta — APPROVED** [review]
LGTM

**Pull request merged**